presto-main/src/main/java/com/facebook/presto/server/security/LdapFilter.java

```diff
@@ -133,7 +138,7 @@ public Principal getUserPrincipal()
                 LOG.debug("LDAP Authentication failed due to invalid credentials");
                 throw Throwables.propagate(e);
             }
-            catch (NamingException e) {
+            catch (Exception e) {
```

Comment:
I would avoid using `Exception` in interfaces. When catching `Exception` it is always easy to swallow `interrupted` flag. Maybe it is not that important it this particular case, but it is always better to follow this best practice.